metersink/lib.py

```diff
@@ -1,23 +1,17 @@
+"""The basic library for the metering tool"""
+
 import configparser
 import logging
 import re
 from datetime import datetime, timedelta
 
-from metersink.output_odoo import (
-    get_odoo_version,
-    odoo_get,
-    get_odoo_user_id,
-    odoo_create,
-    odoo_update,
-)
 from metersink.output_textfile import output_file
```

Comment:
Why this import? It's not used inside this module, and the api module should import this directly, paralleling the import for odoo. The way it is now, it's kind of messy.